src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -824,6 +825,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         const auto ptr = args.Pointer();
         const auto point = args.GetCurrentPoint(*this);
 
+        InputPane::GetForCurrentView().TryShow();

```

Comment:
Does `_GotFocusHandler` not get fired automatically after this one? I'd think you'd only need it in the GotFocus one.

Reply:
Nope, it doesn't get fired if we already have focus. This call is here for the case where terminal is in focus, but the user closed the keyboard.

Reply:
Okay good enough for me. (I'd appreciate adding this as a comment in code, but you do you)